static/app/components/smartSearchBar/index.tsx

```diff
@@ -522,19 +524,13 @@ class SmartSearchBar extends React.Component<Props, State> {
       return;
     }
 
-    if (key === 'Enter') {
-      if (!useFormWrapper && !isSelectingDropdownItems) {
-        // If enter is pressed, and we are not wrapping input in a `<form>`,
-        // and we are not selecting an item from the dropdown, then we should
-        // consider the user as finished selecting and perform a "search" since
-        // there is no `<form>` to catch and call `this.onSubmit`
```

Comment:
Always have to do this now, pressing enter on a `textarea` doesn't trigger an `onSubmit`